vars/filebeat.groovy

```diff
@@ -78,14 +84,29 @@ def start(Map args = [:]) {
     output: output,
     config: config,
     image: image,
```

Comment:
you have removed the parameter workdir from the JSON file

Reply:
I noticed the `workdir` element in the JSON file is never read. In the `stop` method, we read the JSON file into the `stepConfig` variable. From that object, we only read `timeout`, `archiveOnlyOnFail`, `id` and `output`